pkg/consts/overcommit.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+// const variables for node annotations about overcommit ratio
+const (
+	NodeAnnotationCPUOvercommitRatioKey    = "katalyst.kubewharf.io/CPU_overcommit_ratio"
+	NodeAnnotationMemoryOvercommitRatioKey = "katalyst.kubewharf.io/memory_overcommit_ratio"
+
+	NodeAnnotationNativeCapacityCPUKey       = "katalyst.kubewharf.io/native_capacity_CPU"
```

Comment:
Maybe it would be better to name it `katalyst.kubewharf.io/original_capacity_cpu` or `katalyst.kubewharf.io/real_capacity_cpu`?